Formula/supersm.rb

```diff
@@ -0,0 +1,24 @@
+class Supersm < Formula
+  desc "Super Symlink Manager"
+  homepage "https://github.com/peeweep/supersm"
+  url "https://github.com/peeweep/supersm/archive/0.5.2.tar.gz"
+  sha256 "299699209c2af0f91edd659578116dbaac0aa0cd3bd0ad55be6571350fd448da"
+  license "MIT"
+
+  livecheck do
+    url :homepage
+  end
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    assert_match "#{version} \n", shell_output("#{bin}/supersm --version", 1)
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula